src/main/docker/Dockerfile

```diff
@@ -22,7 +22,7 @@ FROM ${DOCKER_HUB_PUBLIC}/cafapi/prereleases:buildenv-dotnet-1.1.0-SNAPSHOT
 # Copy in zypper repo key
 COPY dev_zypper_repo_key.pub .
 
-RUN zypper -n addrepo https://download.opensuse.org/repositories/home:zkrd2:mono/15.4/home:zkrd2:mono.repo && \
+RUN zypper -n addrepo --no-gpgcheck https://download.opensuse.org/repositories/home:/lemmy04/15.4/home:lemmy04.repo && \
```

Comment:
I couldn't get the signing key to be auto accepted so I had to turn off gpgchecks, is this ok? 

![image](https://user-images.githubusercontent.com/93921604/203551400-0dc0bdbe-ce63-453d-90b7-a1b0bf38a321.png)